### ISSUE TITLE
Repeating Events

### ISSUE THREAD

**Issue by markc6275** [iBurnApp/iBurn-iOS#172]
Hi, Thanks again for the app. It is very helpful.
Is there an easy way to add all repeating events? Maybe sort by event name so all repeating events can selected at the same time. Then you could easily see that an event repeats (or not) and choose to go at another time if it conflicts with something else.
Thanks!
-Mark

**Comment by chrisballinger** [comment]
good idea! the easiest way to see this is currently to go to the camp that's hosting the event, and then see all events hosted by the camp
